Formula/smartdns.rb

```diff
@@ -0,0 +1,49 @@
+class Smartdns < Formula
+  desc "Rule-based DNS server for fast IP resolution, DoT/DoH/DoQ supported"
+  homepage "https://github.com/mokeyish/smartdns-rs"
+  url "https://github.com/mokeyish/smartdns-rs/archive/refs/tags/0.3.1.tar.gz"
+  sha256 "3a11e1b1c6ffc33d1120b6cec516d19a90375be283c114771cf950e2b364215d"
+  license "GPL-3.0-or-later"
+  head "https://github.com/mokeyish/smartdns-rs.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
They don't seem to have any pre-releases so this isn't needed

Reply:
Then we can also add this later.

Reply:
Ok, I remove it.